service/storage_service.cc

```diff
@@ -1227,6 +1227,12 @@ class topology_coordinator {
         slogger.trace("raft topology: start CDC generation publisher fiber");
 
         while (!_as.abort_requested()) {
+            co_await utils::get_local_injector().inject_with_handler("cdc_generation_publisher_fiber", [] (auto& handler) -> future<> {
+                slogger.info("raft toplogy: CDC generation publisher fiber sleeps after injection");
+                co_await handler.wait_for_message(std::chrono::steady_clock::now() + std::chrono::minutes{2});
```

Comment:
Very optimistic to assume that you'll manage to bootstrap 3 nodes in 2 minutes on our great CI machines with all these concurrent tests :) And let's not even talk about ARM.

Let's increase the timeout to 5 minutes. And... let's mark the two tests dev-mode only.

Reply:
Done.